remove status-app from membership

johnduffell opened the pull request:
This PR stops status app deploying to membership.  I will also remove status app from the membership AWS account.

This means we don't have to keep paying for it and maintaining it!

rupertbates reviewed (approved):
👍

rtyley commented:
Fair play - we still use it on the Ophan team, I guess no-ones using it in the Membership account? I've just opened a PR to update status-app to latest Play, sbt, etc: #94

johnduffell commented:
thanks @rtyley yes every time we ask around noone is using it, only Paul B mentioned it's useful to look at the SQS queues.

I also noticed just now that the AMI has stopped baking, perhaps because it's on an old ubuntu and has some kind of python/pip incompatibility.  So if you're still using it then it may be affecting you.

johnduffell commented:
in membership account I have just
- deleted the stack
- removed the DNS validation record
- deleted the SSL certificate
- stopped amigo from pushing the baked AMIs into the membership account

Hopefully that's everything cleaned up but I'm happy to believe there are other things lying around there still to clean.
